apps/cowswap-frontend/vite.config.ts

```diff
@@ -132,9 +132,6 @@ export default defineConfig(({ mode }) => {
           manualChunks(id) {
             if (id.includes('@1inch')) return '@1inch'
             if (id.includes('@amplitude')) return '@amplitude'
-            if (id.includes('@cowprotocol')) return '@cowprotocol'
-            if (id.includes('@ethersproject')) return '@ethersproject'
-            if (id.includes('@metamask') || id.includes('elliptic')) return '@metamask'
```

Comment:
I had to remove these 3 chunks as the app would not load :/

Reply:
😢 feel fragile! is not from this PR so fine, but we will need to understand better later on how to fine-tune our depenendencies

Reply:
I think the next step to improve on this is to identify and actively make use of dynamically loading of deps and parts of the app.